en/simulation/multi_vehicle_simulation_gazebo.md

```diff
@@ -46,6 +49,53 @@ https://youtu.be/aEzFKPMEfjc
 https://youtu.be/lAjjTFFZebI
 {% endyoutube %}
 
+
+
+## Multiple Vehicles with Gazebo based on RTPS {#with_rtps}
+
```

Comment:
Why would someone want to do this? As a reader I see this option and I think. "that looks like more steps - what is the benefit/why do it this way".

Reply:
I think it is so complicated. However, it will be needed if ROS2/RTPS is more used.

Reply:
Cool. @Jaeyoung-Lim Can you perhaps summarise the reason to do this? I just don't like options where it isn't obvious the benefit or when you need to choose them.

Note, the new PR is https://github.com/PX4/Devguide/pull/1076